azure/functions/decorators/function_app.py

```diff
@@ -419,6 +443,20 @@ def decorator():
 
         return wrap
 
+    def function_name(self, name: str,
```

Comment:
The issue about adding it here is that now it's a trigger which it's not. Thats the reason a new api was created to add decorators which were not triggers or binding.